FFmpegInterop/Tracing.h

```diff
@@ -0,0 +1,106 @@
+//*****************************************************************************
+//
+//	Copyright 2019 Microsoft Corporation
+//
+//	Licensed under the Apache License, Version 2.0 (the "License");
+//	you may not use this file except in compliance with the License.
+//	You may obtain a copy of the License at
+//
+//	http ://www.apache.org/licenses/LICENSE-2.0
+//
+//	Unless required by applicable law or agreed to in writing, software
+//	distributed under the License is distributed on an "AS IS" BASIS,
+//	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//	See the License for the specific language governing permissions and
+//	limitations under the License.
+//
+//*****************************************************************************
+
+#pragma once
+
+namespace winrt::FFmpegInterop::implementation
+{
+	TRACELOGGING_DECLARE_PROVIDER(g_FFmpegInteropProvider);
+
+	// TODO: Replace these macros with WIL TraceLogging macros if they're ever made public

```

Comment:
I know this work probably isn't planned, just remove the TODO. E.g. "These macros should be replaced with WIL TraceLogging macros if they're ever made public."

Reply:
Removed.